upbound_uxp/hatch.toml

```diff
@@ -0,0 +1,4 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.8"]
```

Comment:
I believe the CI only supports python 3.9 now.

Reply:
Updated.